addons-l10n/en/find-bar.json

```diff
@@ -1,4 +1,9 @@
 {
   "find-bar/find-placeholder": "Find (Ctrl+F)",
-  "find-bar/complex-broadcast": "(expression)"
+  "find-bar/complex-broadcast": "(expression)",
+  "find-bar/var-local": "variable {name}",
+  "find-bar/var-global": "VARIABLE {name}",
+  "find-bar/list-local": "list {name}",
+  "find-bar/list-global": "LIST {name}",
```

Comment:
Self TODO: add translation notes here for languages that don't distinguish lowercase and uppercase. They could explicitly translate the strings as "local variable" or "global variable" so that the distinction remains.

Reply:
Please tag me if I haven't looked into this after a few days

Reply:
@WorldLanguages Here's your reminder.